source/features/embed-gist-via-iframe.tsx

```diff
@@ -0,0 +1,37 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const embedViaScript = select('.select-menu-item')!;
+	const embedViaIframe = embedViaScript.cloneNode(true) as HTMLElement;
+
+	// Modify description to distinguish the two items
+	// This is also consistent with the existing "Clone via HTTPS" and "Clone via SSH" items
+	const viaScript = 'Embed via <script>';
+	select('[data-menu-button]')!.textContent = viaScript;
+	select('.select-menu-item-heading', embedViaScript)!.textContent = viaScript;
+	select('.description', embedViaScript)!.textContent = 'Embed this gist in your website via <script>.';
+
+	// Remove analytics attributes
+	embedViaIframe.removeAttribute('data-hydro-click');
+	embedViaIframe.removeAttribute('data-hydro-click-hmac');
+
+	// Set required content
+	embedViaIframe.setAttribute('aria-checked', 'false');
+	embedViaIframe.setAttribute('value', embedViaIframe.getAttribute('value')!.replace(/script/g, 'iframe').replace('.js', '.pibb'));
```

Comment:
Properties are more readable than attributes here

```suggestion
	embedViaIframe.value = embedViaIframe.value!.replace(...)
```

Also this fails if the username or domain (for GH Enterprise) contains `script` or `.js` anywhere. It needs to match the HTML tag more precisely

Reply:
> Properties are more readable than attributes here

That will fail the type check as it is marked as `HTMLElement` 😅 well I should have changed it to `HTMLButtonElement`

> Also this fails if the username or domain (for GH Enterprise) contains `script` or `.js` anywhere. It needs to match the HTML tag more precisely

Thanks for reminding me, should adopt the previously used method: extract URL from `location`